cmd/configure/configure.go

```diff
@@ -142,15 +103,31 @@ func newConfigureCommand() *cobra.Command {
 			return fmt.Errorf("unable to instantiate configuration from environment variables: %w", err)
 		}
 
+		// Populate configuration from flags (if set).
+		if flags.Host != "" {
+			cfg.Host = flags.Host
+		}
+		if flags.Profile != "" {
+			cfg.Profile = flags.Profile
+		}
+
+		// Verify that the host is valid (if set).
+		if cfg.Host != "" {
+			err = validateHost(cfg.Host)
+			if err != nil {
+				return err
+			}
+		}
+
 		ctx := cmd.Context()
 		interactive := cmdio.IsInTTY(ctx) && cmdio.IsOutTTY(ctx)
-		var fn func(*cobra.Command, context.Context, *config.Config) error
+		var fn func(*cobra.Command, *configureFlags, *config.Config) error
```

Comment:
Not your change here, but what was so bad about
```go
if interactive {
  err = configureInteractive(cmd, &flags, &cfg)
} else {
  err = configureNonInteractive(cmd, &flags, &cfg)
}
```
?

Reply:
No idea, changed to your suggestion.